pandas/tests/extension/base/methods.py

```diff
@@ -105,6 +105,31 @@ def test_factorize_equivalence(self, data_for_grouping, na_sentinel):
         tm.assert_numpy_array_equal(l1, l2)
         self.assert_extension_array_equal(u1, u2)
 
+    def test_fillna_copy_frame(self, data_missing):
+        arr = data_missing.take([1, 1])
+        df = pd.DataFrame({"A": arr})
+
+        filled_val = df.iloc[0, 0]
+        result = df.fillna(filled_val)
+
+        assert df.values.base is not result.values.base
+        assert df.A._values is arr
```

Comment:
This can be removed I think. That's asserting something different.

Reply:
Done! Thanks.